File: ZoneSentry/ClientApp/src/api/models/SearchFilters.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { RealtyStatus } from './RealtyStatus';

export type SearchFilters = {
    realtyStatus?: RealtyStatus;
    city?: string | null;
    areaMin?: number | null;
    areaMax?: number | null;
    priceMin?: number | null;
    priceMax?: number | null;
};

File: ZoneSentry/Models/SearchFilters.cs
namespace ZoneSentry.Models;

public class SearchFilters
{
    public RealtyStatus? RealtyStatus { get; set; }
    public string? City { get; set; }
    
    public int? AreaMin { get; set; }
    public int? AreaMax { get; set; }
    public int? PriceMin { get; set; }
    public int? PriceMax { get; set; }
}